shell/common/shell.cc

```diff
@@ -605,6 +607,14 @@ void Shell::RunEngine(
             if (run_result == flutter::Engine::RunStatus::Failure) {
               FML_LOG(ERROR) << "Could not launch engine with configuration.";
             }
+
+            // Enable platform channels for background isolates.
+            std::shared_ptr<DartIsolate> root_isolate =
```

Comment:
It would be more consistent with the engine's design to have the `RuntimeController` get the `PlatformMessageHandler` through the `RuntimeDelegate` and then set it on the root `DartIsolate`

The `Shell` operates at a higher layer and does not interact directly with `DartIsolate` instances

Reply:
done.